qemu/tests/cfg/cpu_device_hotpluggable.cfg

```diff
@@ -89,6 +97,10 @@
             numa_memdev_node1 = mem-mem1
             backend_mem_mem0 = memory-backend-ram
             backend_mem_mem1 = memory-backend-ram
+            # case include hot-unplug steps
```

Comment:
hot-unplug step is for linux only:
https://github.com/autotest/tp-qemu/blob/683ca755c5b819174e4c3773ecba94bdbd73bb35/qemu/tests/cpu_device_hotpluggable_with_numa.py#L106
https://github.com/autotest/tp-qemu/blob/683ca755c5b819174e4c3773ecba94bdbd73bb35/qemu/tests/cpu_device_hotpluggable_with_numa.py#L124

Reply:
> hot-unplug step is for linux only:
> 
> https://github.com/autotest/tp-qemu/blob/683ca755c5b819174e4c3773ecba94bdbd73bb35/qemu/tests/cpu_device_hotpluggable_with_numa.py#L106
> 
> 
> https://github.com/autotest/tp-qemu/blob/683ca755c5b819174e4c3773ecba94bdbd73bb35/qemu/tests/cpu_device_hotpluggable_with_numa.py#L124

@yanan-fu  yes, and also only x86_64 will cover ovmf test so the limition only works for x86 test. so it won't affect the other case I think.

Reply:
@nanliu-r  What about windows ?    It is linux only limitation but be applied for not only linux.

Reply:
> @nanliu-r What about windows ? It is linux only limitation but be applied for not only linux.

You are right, updated and thanks.